app/paths.js

```diff
@@ -156,13 +156,18 @@ module.exports = {
     addMetadata: '/create-payment-link/add-reporting-column',
     editMetadata: '/create-payment-link/add-reporting-column/:metadataKey',
     deleteMetadata: '/create-payment-link/add-reporting-column/:metadataKey/delete',
-    manage: '/create-payment-link/manage',
-    disable: '/create-payment-link/manage/disable/:productExternalId',
-    delete: '/create-payment-link/manage/delete/:productExternalId',
-    edit: '/create-payment-link/manage/edit/:productExternalId',
-    editInformation: '/create-payment-link/manage/edit/information/:productExternalId',
-    editReference: '/create-payment-link/manage/edit/reference/:productExternalId',
-    editAmount: '/create-payment-link/manage/edit/amount/:productExternalId',
+    manage: {
+      edit: '/create-payment-link/manage/edit/:productExternalId',
+      managePage: '/create-payment-link/manage',
+      disable: '/create-payment-link/manage/disable/:productExternalId',
+      delete: '/create-payment-link/manage/delete/:productExternalId',
+      editInformation: '/create-payment-link/manage/edit/information/:productExternalId',
+      editReference: '/create-payment-link/manage/edit/reference/:productExternalId',
+      editAmount: '/create-payment-link/manage/edit/amount/:productExternalId',
+      addMetadata: '/create-payment-link/manage/:productExternalId/add-reporting-column',
+      editMetadata: '/create-payment-link/manage/:productExternalId/add-reporting-column/:metadataKey',
+      deleteMetadata: '/create-payment-link/manage/:productExternalId/add-reporting-column/:metadataKey/delete'
+    },
     metadata: {
```

Comment:
Should this come under `manage` as well? Or are these routes being removed soon anyway?

Reply:
These are being removed anyway as part of `PP-7508`, they're needed in the meantime for the backend functionality